libraries/Microsoft.Bot.Builder.BotFramework/BotFrameworkBotServer.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Servers;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.BotFramework
+{
+    public class BotFrameworkBotServer : BotServer
```

Comment:
Given how central this class is going to be, I'd love to put more consideration in the name. BotFrameworkBotServer sounds really awkward. Could we think of something more concise and sticky?